src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -164,6 +172,12 @@ public void islDefaultRulesDelete(Endpoint source, Endpoint destination) {
                 new SpeakerRulesIslRemoveCommand(keyFactory.next(), source, destination)));
     }
 
+    @Override
+    public void auxiliaryPollUpdateRequest(Endpoint endpoint, boolean enableAuxiliaryPoll) {
+        WatchListCommand command = new WatchListAuxiliaryPollUpdateCommand(endpoint, enableAuxiliaryPoll);
+        emit(STREAM_POLL_ID, makePollTuple(command));
```

Comment:
Add a comment describing why we do not link produced tuple to the current tuple i.e. why we do not use the anchor. Without the comment, someone can think this is a typo/mistake.

Reply:
Added.